xcube/core/store/fs/registry.py

```diff
@@ -25,7 +25,9 @@
 
 from .accessor import FsAccessor
 from .accessor import FsDataAccessor
-from .impl.dataset import DatasetNetcdfFsDataAccessor
+from .impl.cog import MultiLevelDatasetGeoTiffFsDataAccessor
+from .impl.dataset import DatasetNetcdfFsDataAccessor, \
+    DatasetGeoTiffFsDataAccessor
```

Comment:
We put each import in a single line (exception: python standard lib)
```suggestion
from .impl.dataset import DatasetGeoTiffFsDataAccessor
from .impl.dataset import DatasetNetcdfFsDataAccessor
```